### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by Kiran-Serolla**
Added week2 homework..

### CODE REVIEW

File: javascript/javascript2/week2/movies.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
+const movies = [
```
Comment (florinmaticu) [inline javascript/javascript2/week2/movies.js:1]: For the movies exercise, please not include all the movies array. is not okay...you should not make your own movies variable. Instead include javascript2/week2/homework/movies.js into your .html file

File: javascript/javascript2/week2/movies.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+// movies made betweem 1980 and 1989
+
+const movieCount = movies.filter(
+  (movie) => movie.year >= 1980 && movie.year <= 1989
+);
+console.log(movieCount.length);
+
+// Calculating rating
+
+const arrayWithNewTag = movies.map((ratingValue) => {
```
Comment (florinmaticu) [inline javascript/javascript2/week2/movies.js:178]: The variable name is not okay
```suggestion
const moviesWithRating= movies.map((ratingValue) => {
```

Comment (Kiran-Serolla) [reply javascript/javascript2/week2/movies.js]: Changed the variable names

File: javascript/javascript2/week2/movies.js (hunk 1 above)
Comment (Kiran-Serolla) [reply javascript/javascript2/week2/movies.js]: Hi Florin,
I tried several ways to include movies array from external file but it isnt working.... but I still have this modified vesion with external movies array.

**Pull request merged**